seed-bank-dynamics/scripts/README.md

```diff
@@ -0,0 +1,13 @@
+# Data set on free-living birds in Sweden
```

Comment:
```suggestion
# Data set on Dung Beetle activity in rainforests
```

Reply:
I set up the whole second folder with data from Zenodo before I realised that the data set for the seed beetles where also available from Zenodo. Hence the change of name of the second resource and the fact that I've created the second data file.